source/settings/plugin/imap-compress-plugin.rst

```diff
@@ -0,0 +1,47 @@
+.. _plugin-imap-compress:
+
+====================
+IMAP COMPRESS plugin
+====================
+
+The goal of the extension is to reduce the bandwidth usage of IMAP.
+
+Configuration:
+
+.. code-block:: none
+
+  #mail_plugins = $mail_plugins zlib # Required for v2.1 and older only
+
+  protocol imap {
+    mail_plugins = $mail_plugins imap_zlib
+  }
+
+  plugin {
+    ## NOTE! Setting was called imap_zlib_compression_level before 2.3.15
+    ## imap_compress_<algorithm>_level = value
+    ##
+    # imap_compress_deflate_level = 6
+  }
+
+
+Settings
+========
+
+.. _setting-plugin_imap_compress_level:
+
+``imap_compress_<algorithm>_level``
+----------------------------------------
+
+.. versionchanged:: v2.3.15
```

Comment:
This one has "v" while others don't. It's not entirely consistent everywhere though .. I wonder if we could somehow enforce it?..

Reply:
not sure how, would need some tool.